tutorials/tts/FastPitch_MixerTTS_Training.ipynb

```diff
@@ -1,612 +1,611 @@
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "from nemo.collections.tts.models.base import SpectrogramGenerator\n",
-    "from nemo.collections.tts.models import FastPitchModel, MixerTTSModel\n",
-    "\n",
-    "from matplotlib.pyplot import imshow\n",
-    "from matplotlib import pyplot as plt\n",
-    "%matplotlib inline"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "cdf4aee7",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# Let's see what pretrained models are available for FastPitch and Mixer-TTS\n",
-    "print(\"FastPitch pretrained models:\")\n",
-    "print(FastPitchModel.list_available_models())\n",
-    "print(\"=====================================\")\n",
-    "print(\"Mixer-TTS pretrained models:\")\n",
-    "print(MixerTTSModel.list_available_models())"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "298704c4",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# We can load the pre-trained FastModel as follows\n",
-    "pretrained_model = \"tts_en_fastpitch\"\n",
-    "spec_gen = FastPitchModel.from_pretrained(pretrained_model)\n",
-    "spec_gen.eval();"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "c18181ff",
-   "metadata": {
-    "scrolled": true
-   },
-   "outputs": [],
-   "source": [
-    "# In the same way, we can load the pre-trained Mixer-TTS model as follows\n",
-    "pretrained_model = \"tts_en_lj_mixertts\"\n",
-    "spec_gen = MixerTTSModel.from_pretrained(pretrained_model)\n",
-    "spec_gen.eval();"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "fb41b646",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "assert isinstance(spec_gen, SpectrogramGenerator)\n",
-    "\n",
-    "if isinstance(spec_gen, FastPitchModel):\n",
-    "    tokens = spec_gen.parse(str_input=\"Hey, this produces speech!\")\n",
-    "else:\n",
-    "    tokens = spec_gen.parse(text=\"Hey, this produces speech!\")\n",
-    "\n",
-    "spectrogram = spec_gen.generate_spectrogram(tokens=tokens)\n",
-    "\n",
-    "# Now we can visualize the generated spectrogram\n",
-    "# If we want to generate speech, we have to use a vocoder in conjunction to a spectrogram generator.\n",
-    "# Refer to the Inference_ModelSelect notebook on how to convert spectrograms to speech.\n",
-    "imshow(spectrogram.cpu().detach().numpy()[0,...], origin=\"lower\")\n",
-    "plt.show()"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "54ec3c5e",
-   "metadata": {},
-   "source": [
-    "# Preprocessing"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "7ef87e31",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "from nemo.collections.tts.torch.data import TTSDataset\n",
-    "from nemo_text_processing.text_normalization.normalize import Normalizer\n",
-    "from nemo_text_processing.g2p.modules import EnglishG2p\n",
```

Comment:
I believe `main` has the correct path here (i.e. `nemo_text_processing.g2p.modules`) and is more up-to-date.

Reply:
Thanks for the info! i will fix the rules for yaml and tutorial, that keep main in main branch.

Reply:
fixed.